model/schema/reference.yaml

```diff
@@ -199,122 +191,125 @@ slots:
     multivalued: false
     range: string
     exact_mappings:
       - WIKIDATA_PROPERTY:P304
     
   plain_language_abstract:
-    description: Plain language abstract
+    description: >-
+      Plain language abstract.
+      Only relevant for PubMed references.
     multivalued: false
     domain: Reference
     range: string
 
   pubmed_abstract_languages:
-    description: Languages for the abstract
+    description: >-
+      Languages for the abstract.
+      Only relevant for PubMed references.
     multivalued: true
     domain: Reference
     range: string
 
   pubmed_publication_status:
-    description: status of the publication
+    description: >-
+      Status of the publication at PubMed.
+      Only relevant for PubMed references.
     multivalued: false
     domain: Reference
     range: pubmed_publication_status_enum
 
   abstract:
     description: >-
       The author summary of the publication.
-    multivalued: false
-    domain: Reference
-    range: string
-
-  citation:
-    description: >-
-      A short form way of referring to a publication, typically consisting of
-      the first author's name, publication date, and journal (title, vol, pages).
+      From PubMed otherwise from Mod or manual reference creation.
     multivalued: false
     domain: Reference
     range: string
 
   issue_name:
+    description: >-
+      The issue name.
+      From PubMed otherwise from Mod or manual reference creation.
     multivalued: false
     domain: Reference
     range: string
 
   category:
+    description: >-
+      The alliance category type.  Only relevant at Alliance.
     multivalued: false
     domain: Reference
     range: reference_category_enum
 
   keywords:
+    description: >-
+      Keywords tagging a publication.  Aggregation of PubMed and ZFIN,
+      editable at Alliance.
     multivalued: true
-    description: keywords tagging a publication
     range: string
   
   language:
+    description: Language of the reference.  Aggregation of PubMed and FlyBase,
+      editable at Alliance.
     multivalued: false
-    description: Language of the reference
     domain: Reference
     range: string
 
   merged_into_id:
-    multivalued: false
     description: ID that used to refer to this reference
+    multivalued: false
     domain: Reference
     range: uriorcurie
 
-  summary:
-    description: >-
-    domain: Reference
-    range: string
-
-  copyright_date:
-    description: >-
-    domain: Reference
-    range: date
-
   authors:
+    description: >-
+      Ordered author entities for this publication.  An Author is associated
+      with only one publication.  A Person can be associated with multiple 
+      publications.
     multivalued: true
     domain: Reference
     range: AuthorReference
 
   corresponding_author:
-    description: >-
-    range: Reference
+    description: Indicates if the author is a corresponding author.
+    range: boolean
     domain: Reference
```

Comment:
1. Should add `range: AuthorReference`?
2. Should add `multivalued: true`? Usually only one corresponding author, but are there edge cases where a paper has many corresponding authors?